src/views/PlanTargets.vue

```diff
@@ -0,0 +1,198 @@
+            :lon="s.longitude"
+            :value="s.siteoffset"
+            >{{s.name}}</option>
+            <option lat="" lon="" value="X">None</option>
+          </b-select>
+        </b-field
+        <br>
+        <div class="field has-addons">
+          <p class="control is-expanded">
+            <b-field label="Latitude">
+              <b-input type="text" id="lat1" v-model="user.lat1"/>
+            </b-field>
+          </p>
+          <p class="control is-expanded">
+            <b-field label="Longitude">
+              <b-input type="text" id="lon1" v-model="user.lon1"/>
+            </b-field>
+          </p>
+        </div>
+        <div class="field has-addons">
+          <p class="control">
+            <b-field label="Date/Time">
+              <b-datetimepicker 
+                id="dateobs" 
+                v-model="user.dateobs"
+                :timepicker="{ incrementMinutes:30 }"
+                required 
+                inline />
+            </b-field>
+          </p>
+          </p>
+        </div>
+      <b-field grouped>
+        <div v-if="user.lcltime !== 'X'">
+          <b-radio name="tzinfo" id="tzinfo" native-value="my" v-model="user.tzinfo" required>My timezone</b-radio>
+          <b-radio name="tzinfo" id="tzinfo" native-value="utc" v-model="user.tzinfo" required> UTC</b-radio>
+          <b-radio name="tzinfo" id="tzinfo" native-value="lcl" v-model="user.tzinfo">Local timezone</b-radio>
+        </div>
+        <div v-else>
+          <b-radio name="tzinfo" id="tzinfo" native-value="my" v-model="user.tzinfo" required>My timezone</b-radio>
+          <b-radio name="tzinfo" id="tzinfo" native-value="utc" v-model="user.tzinfo" required> UTC</b-radio>
+        </div>
+      </b-field>
+        <div class="the-button">
+          <b-field class="buttons">
+            <b-button @click="submitForm" type="submit">Find Easy Targets</b-button>
+          </b-field>
+        </div>
+    </form>
+  </div>
+  <div class="results" id="results">
+    <div class="columns">
+      <div v-for="target in targlist" :target="target" :key="target.name">
+        <TargetCard :target="target" />
+      </div>
+    </div>
+  </div>
+</section>
+</template>
+
+<script>
+import Vue from 'vue';
+import axios from 'axios';
+import moment from 'moment';
+import TargetCard from '@/components/TargetCard.vue';
+import list from '../../public/data/easytargets.json';
+import helpers from '@/utils/helpers';
+
+export default {
+  name: 'PlanTargets',
+  components: {
+    TargetCard,
+  },
+  data() {
+    return {
+      site_info: {},
+      easylist: list,
+      target: {},
+      targlist: '',
+      user: {
+        lat1: '',
+        lon1: '',
+        dateobs: '',
+        tzinfo: 'my',
+        offset: '',
+        lcltime: '',
+        lcloffset: '',
+      },
+    };
+  },
+  created: function() {
+    const url = "https://api.photonranch.org/api/all/config"
+    axios.get(url).then(response => {
```

Comment:
At one point, I had left the default site as "None" (which fixes the issue about a required selection when inputting your own latitude/longitude), but that wouldn't be great for "load page, click button, get results" - which do you think would be preferable, that or setting mrc (e.g.) as a default?

Reply:
Hmm, how about this:

If a site is selected from the dropdown, add the `disabled` property to the coordinate fields so the value is displayed but users can't change it.

Either with a radio button toggling 'site vs manual coordinates' or with an extra value in the dropdown (like your 'none' option), users can specify a manual coordinate option which re-enables the coordinate fields for user input. 

I think a default value of MRC for a location would be a safe option, but I'll let you make the final call.

Reply:
Fixed all the other stuff but I haven't been able to find a way to make the datetimepicker accept a default value - the usual way of just setting an initial value doesn't seem to work (either at definition or using Vue.set), and I can't find any suggestions online about how to do so.

It's possible to set an initial value for "dateobs" and make the datetimepicker not required, which will still allow for a user to load page, click button, get results, but it might be unintuitive that the date/time isn't visibly picked - any thoughts?

Reply:
Never mind - think I got it working!